proposals/lip-0053.md

```diff
@@ -732,7 +735,8 @@ def execute(trs: Transaction) -> None:
     for ccmBytes in crossChainMessages:
         ccmID = sha256(ccmBytes)
         # Set ccmID (instead of the transaction ID) as default topic to all events emitted in apply.
-        defaultEventsTopic = ccmID
+        # The default topic is added automatically by emitEvent.
+        eventsDefaultTopic = ccmID
```

Comment:
Same as above: could potentially removed.

Reply:
Notice that ccmID changes during the loop, so we can't remove this

Reply:
right, I missed this